### ISSUE TITLE
Deposit transaction displays a number for Quantity

### ISSUE THREAD

**Issue by PatrickLuhmann** [PatrickLuhmann/Couatl3#12]
Quantity has no meaning for a Deposit transaction. This field in the transaction list view should be blank. If there is a number there, even if it is 0, the user might think that there can be a quantity associated with a deposit, even though this is not correct.

**Comment by PatrickLuhmann** [comment]
Also looking at Security because it also has no meaning for a Deposit/Withdrawal.

**Comment by PatrickLuhmann** [comment]
The key seems to be that the control was not being notified of the changes. I had to go down to the bottom of the object hierarchy and raise notifications for the individual properties of Transaction_VM, not just TheTransaction. This also required that the View bind to Quantity, not TheTransaction.Quantity.

I made Quantity nullable because the control is smart enough to simply leave the view blank if the value is null. If this didn't work, then I think I would have needed to create a converter or something similar, which would have been overkill for what I want to do.
